applications/ContactStructuralMechanicsApplication/python_scripts/auxiliar_methods_solvers.py

```diff
@@ -127,6 +109,17 @@ def  AuxiliarValidateSettings(solver):
     default_settings.RecursivelyAddMissingParameters(solver.settings)
     solver.settings.RecursivelyValidateAndAssignDefaults(default_settings)
 
+    # Common settings
+    if not solver.settings["clear_storage"].GetBool():
+        KM.Logger.PrintInfo("Clear storage", "Storage must be cleared each step. Switching to True")
+        solver.settings["clear_storage"].SetBool(True)
+    if not solver.settings["reform_dofs_at_each_step"].GetBool():
+        KM.Logger.PrintInfo("Reform DoFs", "DoF must be reformed each time step. Switching to True")
+        solver.settings["reform_dofs_at_each_step"].SetBool(True)
+    if solver.settings["use_computing_model_part"].GetBool():
+        KM.Logger.PrintInfo("Using Computing-ModelPart. Switching to False")
+        solver.settings["use_computing_model_part"].SetBool(False)
```

Comment:
but here you do the same don't you?

Either way, you know better :)

Reply:
This works...(I need to fully revisit my submodelpart structure to revert those changes, and that it is going to be painful)

Reply:
ok